core/backend/src/EntityReference.ts

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Schema
+ */
+
+import { EntityReferences as BentleyEntityReferences, ConcreteEntityTypes, ElementAspectProps, ElementProps, EntityReference, ModelProps } from "@itwin/core-common";
+import { Id64String } from "@itwin/core-bentley";
+import type { Entity } from "./Entity";
+import type { Model } from "./Model";
+import type { Element } from "./Element";
+import type { ElementAspect } from "./ElementAspect";
+import type { Relationship, RelationshipProps } from "./Relationship";
+import * as assert from "assert";
+
+// re-export so consumers don't need to manually import the basic types we are extending
```

Comment:
Why?

Reply:
so people don't need to import from two separate packages `EntityReference` and `EntityReferences` when in a backend context. It doesn't really matter so I can just remove the re-export and split existing imports that use the reexport

Reply:
Please do - relying on internals of other package's lib dirs is flaky (for example, CommonJs assumed here).